cpp/include/cudf/quantiles.hpp

```diff
@@ -32,16 +32,20 @@ namespace experimental {
  * @param[in] interp       Strategy used to interpolate between the two values
  *                         on either side of the desired quantile.
  * @param[in] column_order Indicates the sortedness of the column.
- *
+ * @param exact If true, returns doubles. If false, returns same type as input.
+
  * @returns Value of the desired quantile, or null if the column has no valid
```

Comment:
Since this is no longer returning a scalar, maybe more explanation to tell what is in the returned column? Is it a column with a single value in it? Maybe add some examples?

Reply:
Once can request values at more than one quantiles from a column e.g. if vector `q` == `{0,1}` then this should return the first and last element. The doc needs to be:
```suggestion
 * @returns Values at the desired quantiles, contains null elements if the column has no valid
```

Reply:
Documentation updated. Please re-review. :)

Reply:
better than my suggestion